guideforms.py

```diff
@@ -37,15 +37,28 @@
     'name': forms.CharField(
         required=True, label='Feature',
         widget=forms.TextInput(attrs={'autocomplete': 'off'}),
-        help_text=('Capitalize only the first letter and the beginnings of '
-                   'proper nouns.')),
+        help_text=
+        ('Capitalize only the first letter and the beginnings of '
+         'proper nouns. '
+         '<a target="_blank" href="'
+         'https://github.com/GoogleChrome/chromium-dashboard/wiki/'
+         'EditingHelp#feature-name">Learn more</a>. '
```

Comment:
```suggestion
         'EditingHelp#feature-name-learn-more">Learn more</a>. '
```

Reply:
Why add that to the URL anchor?

The gihub wiki generates HTML for headings that have the words of the heading in the anchor.  So, the current heading "Feature name" makes an anchor named "feature-name".  I don't know how we would change it to have the "-learn-more" suffix other than by adding visible text to the heading, which does not sound desirable.